examples/Polls API.md

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
+
+# Polls
+
+Polls is a simple web service that allows consumers to view polls and vote in them.
+
+# Polls API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances in the form of the links in the JSON body.
+
+It is recommend to follow the “url” link values or Link headers to get to resources instead of constructing your own URLs to keep your client decoupled from implementation details.
```

Comment:
To address your comments about it being later and not properly portrayed here. Would the following sentence be any better?

> It is recommend to follow the “url” link values or [Link headers](https://tools.ietf.org/html/rfc5988) where applicable to get to resources instead of constructing your own URLs to keep your client decoupled from implementation detail

Adding "where applicable" to hint that it's not always there?